### DISCUSSION TITLE
Cant find container on Amazon

### DISCUSSION THREAD

**Discussion by Snej-Corleone** [jef/streetmerchant#2988] [Help] [answered]
Hello,

After a site update, you can only see if a PlayStation is available on the Amazon Website by clicking "see all Buying Options" . There are a lot of companies who sell the PlayStation there (for extreme prices...) , so I want to filter that the script only sends out an alert when "Amazon" is the official seller. To do this, I changed the amazon-de.ts to the following: 

` inStock: {
      container: 'span.a-size-small.a-color-base',
      text: ['Amazon'],
    },`

My problem is that the script is not able to find this container. If I remove the "container" statement and just search for the text component, the script finds everything. But the "Amazon" key is logically always on the Page, so the script would send an alarm every time. Can anybody tell me what I have to do in order to make this work and just search in the specific container for the keyword?

 Thank you very much!!

**Comment by Mr-Tech-13** [comment, marked as answer]
Try different containers
